plugins/occt/module/vtkF3DOCCTReader.cxx

```diff
@@ -549,6 +596,9 @@ void vtkF3DOCCTReader::PrintSelf(ostream& os, vtkIndent indent)
   os << indent << "AngularDeflection: " << this->AngularDeflection << "\n";
   os << indent << "RelativeDeflection: " << (this->RelativeDeflection ? "true" : "false") << "\n";
   os << indent << "ReadWire: " << (this->ReadWire ? "true" : "false") << "\n";
-  os << indent << "FileFormat: " << (this->FileFormat == FILE_FORMAT::STEP ? "STEP" : "IGES")
+  os << indent << "FileFormat: "
+     << (this->FileFormat == FILE_FORMAT::BREP      ? "BREP"
+            : this->FileFormat == FILE_FORMAT::STEP ? "STEP"
+                                                    : "IGES")
```

Comment:
a switch case on top may be a better approach, especially if we add more of them in the future.